core/chainio/avs_writer.go

```diff
@@ -95,6 +95,17 @@ func (w *AvsWriter) SendAggregatedResponse(batchIdentifierHash [32]byte, batchMe
 	txOpts.NoSend = false
 	i := 0
 
+	// Set Retry config for RespondToTaskV2
+	respondToTaskV2Config := retry.DefaultRetryConfig()
+	respondToTaskV2Config.NumRetries = 0
```

Comment:
This one should have a `MaxNumRetries`

Reply:
I didn't mean the name. I meant that the `NumRetries` should not be `0`.

Reply:
I removed the re-assignment so the default is 3 retries.